android/app/build.gradle

```diff
@@ -114,6 +96,7 @@ dependencies {
   implementation(platform('org.jetbrains.kotlin:kotlin-bom:1.9.21'))
   implementation 'androidx.annotation:annotation:1.7.0'
   implementation 'androidx.appcompat:appcompat:1.6.1'
+  implementation 'androidx.car.app:app:1.4.0-rc01'
```

Comment:
rc02 is available https://developer.android.com/jetpack/androidx/releases/car-app#1.4.0-rc02